src/plone/recipe/zeoserver/tests/test_docs.py

```diff
@@ -28,7 +28,8 @@ def setUp(test):
     install('Twisted', test)
     install('hyperlink', test)
     install('idna', test)
-    dependencies = pkg_resources.working_set.require('ZODB3')
+    install('PyHamcrest', test)
```

Comment:
Where does this dependency come from?

Reply:
It is a new dependency of a more recent `Twisted` version than the one the last time the tests were run.